core/templates/pages/volunteer-page/volunteer-page.component.html

```diff
@@ -0,0 +1,443 @@
+<div class="volunteer-page protractor-test-volunteer-page">
+  <div class="row oppia-volunteer-banner">
+    <div class="oppia-volunteer-banner-content flex-gt-md-55">
+      <h1 class="oppia-volunteer-h1">
+        Join our global team of volunteers
+      </h1>
+
+      <div class="oppia-volunteer-content">
+        <p>
+          Contributing to Oppia is an opportunity to impact the lives of millions of students.
+        </p>
+        <p>
+          It doesn&rsquo;t matter where you come from, what language you speak, or how young or
+          old you are - this is a community project, and all are welcome!
+        </p>
+      </div>
+    </div>
+
+    <div class="hide-xs hide-sm hide-md col-4 flex-45 mt-5">
+      <picture>
+        <img alt="Volunteer page map image - Desktop view" [src]="mapImgUrl">
```

Comment:
Please also provide WeP versions of the images.

Reply:
Ditto for other images in this PR.

Reply:
Done.